plugins/pulp_rpm/plugins/importers/yum/repomd/modules.py

```diff
@@ -103,7 +103,7 @@ def _get_profile_defaults(module):
     profile_defaults = {}
     for stream, defaults in module.peek_profile_defaults().items():
         profile_defaults[stream] = defaults.get()
-    return bson.BSON.encode(profile_defaults)
+    return bson.binary.Binary(bson.BSON.encode(profile_defaults))
```

Comment:
I hope we do not use db data for profiles in pulp-2to3-migration but worth checking.

Reply:
Oh we do and we utf-8 encode https://github.com/pulp/pulp-2to3-migration/blob/master/pulp_2to3_migration/app/plugin/rpm/pulp_2to3_models.py#L657

Reply:
@goosemania I don't see that kwarg as one of the documented ones https://pymongo.readthedocs.io/en/latest/api/bson/index.html

Reply:
https://github.com/pulp/pulp-2to3-migration/pull/410/